extra-devel/hotspot/autobuild/defines

```diff
@@ -0,0 +1,5 @@
+PKGNAME=hotspot
+PKGSEC=devel
+PKGDES="Perf GUI for performance analysis"
+PKGDEP="qt-5 elfutils gettext threadweaver ki18n kconfigwidgets kcoreaddons kitemviews kitemmodels kio solid kwindowsystem knotifications kiconthemes kddockwidgets"
```

Comment:
Wrap long lines, Vim user.

Reply:
Pwease reswolwe this couwuversation senpai!